app/components/inplace-select.js

```diff
@@ -64,5 +66,31 @@ export default Ember.Component.extend(InPlace, {
       newValue = newValue==='null'?null:newValue;
       this.send('changeValue', newValue);
     },
-  }
+  },
+
+  closeEditor: observer('multiEditModeOn', function() {
```

Comment:
InPlace is where shards code for select, text, textarea, etc... gets stored.  I think these methods belong somewhere else.